packages/cli/src/commands/init.ts

```diff
@@ -1,11 +1,48 @@
+import * as open from 'open';
+import { Answers } from 'inquirer';
 import { prompt } from '../client';
 import { promptIntroArray } from './init.consts';
+import { HttpServer } from '../server';
+import { SERVER_PORT, SERVER_HOST, REDIRECT_ROUTH, API_OAUTH_URL } from '../constants';
+
+let answers: Answers;
 
 export async function initCommand() {
   try {
-    await prompt(promptIntroArray);
+    answers = await prompt(promptIntroArray);
+
+    await gitHubOAuth();
   } catch (error) {
     // eslint-disable-next-line no-console
     console.log(error);
   }
 }
+
+async function gitHubOAuth(): Promise<string> {
+  const httpServer: HttpServer = new HttpServer();
```

Comment:
```suggestion
  const httpServer = new HttpServer();
```

Typescript has a strong inference mechanism, so you basically can just infer the type here. Some other examples for inference:
```
// No need to do stringVariable: string here.
const stringVariable = 'Banana';
```

Reply:
Yea you're right, still trying to get used to the ts infer types.